SourceGen/RuntimeData/Commodore/C64-Kernal-ROM.sym65

```diff
@@ -0,0 +1,255 @@
+https://raw.githubusercontent.com/c64cryptoboy/c64_ghidra/main/ghidra_scripts/c64LabelAddrs.py
```

Comment:
Left-over comment?

Reply:
Yup, I'll get rid of that (I'm adapting from labels I created for Ghidra for much the same purpose).

Reply:
fixed